packages/sysutils/lirc/patches/lirc-0100-disable-python.patch

```diff
@@ -41,32 +40,32 @@ index 9f3dd143..9619a6eb 100644
  
  pkgconfigdir            = $(libdir)/pkgconfig
  dist_pkgconfig_DATA     = lirc.pc lirc-driver.pc
-@@ -71,6 +79,7 @@ nobase_header_HEADERS   = include/media/lirc.h \
-                           include/linux/input-event-codes.h
- header_HEADERS          = drivers/irpipe/irpipe.h
+@@ -78,6 +86,7 @@ header_HEADERS          = include/linux/
+ mediadir                = $(includedir)/lirc/media
+ media_HEADERS           = include/media/lirc.h
  
 +if HAVE_PYTHON
  py_pkgdir               = $(pkgdatadir)/python-pkg
  dist_py_pkg_DATA        = python-pkg/setup.py \
                            python-pkg/README.rst
-@@ -134,6 +143,8 @@ all-local:
- 	    $(if $(VERBOSE),,-q) build
+@@ -143,6 +152,8 @@ all-local:
+ 	    $(PYTHON) setup.py $(if $(VERBOSE),,-q) build
  endif
  
 +endif
 +
  install-data-hook:
  	$(SED) -i -e '/^plugindir/s|/usr/lib|$(libdir)|' \
  	    $(DESTDIR)$(lirc_confdir)/lirc_options.conf
-@@ -170,6 +181,7 @@ fix-version: .phony
+@@ -177,6 +188,7 @@ fix-version: .phony
  	$(SED) -i '/DATE/s/=.*/="$(GIT_DATE)"/' $(distdir)/VERSION
  	$(SED) -i '/REFS/s|=.*|="$(GIT_REFS)"|' $(distdir)/VERSION
  
 +if HAVE_PYTHON
  $(abs_builddir)/python-pkg/setup.py:
  	cp -ar $(top_srcdir)/python-pkg  $(abs_builddir)
  	chmod -R u+w  python-pkg
-@@ -193,6 +205,7 @@ python-pkg/lirc/config.py: Makefile  $(abs_builddir)/python-pkg/setup.py
+@@ -202,6 +214,7 @@ python-pkg/lirc/config.py: Makefile  $(a
```

Comment:
^ is this right?

Reply:
this is just for easier orientation and can be trimmed